tutorials/algorithms/11_gradients_framework.ipynb

```diff
@@ -0,0 +1,718 @@
+{
```

Comment:
<p>I suggest a tweak:</p><p> </p><blockquote>Let's say that we want to use one of our Estimator gradients classes, then we need a quantum state [] and a Hamiltonian H acting as an observable. For the Sampler gradients, we just need a quantum state.</blockquote><p> </p><p>The part where it says "Then, the state and the Hamiltonian are wrapped into an object defining the expectation value" is no longer necessary, as this is done inside the estimator.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1463/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Done.</p>